### DISCUSSION TITLE
Does foocus fit for changing realistic photo style to digital phot style?

### DISCUSSION THREAD

**Discussion by adird3** [lllyasviel/Fooocus#2298] [Q&A] [answered]
Hey!
i am usin fooocus and its great.
my question is how can i take a photo of real people, and change the style to anime/digital art/3d/cartoon/etc..
i import the photo, describe it, choosing the style that i want, but the output is always realistic style. and it does not take my face (i am using faceswap)

has anyone trying to do what i did and succeed?

**Comment by docppp** [comment, marked as answer]
use prompt + image input as pyrra canny and use blue pencil model. it should do the trick
